src/containers/Header/index.js

```diff
@@ -154,15 +154,15 @@ const Header = (props) => {
 
   const urlLinkMap = {
     ...STATIC_ENV_LINKS,
-    // TODO: store previous sidechains in cookies, add them here
+    // TODO: store previous custom networks in cookies, add them here
   }
 
   const rippledUrl = getSocketUrl(rippledSocket)
 
-  if (process.env.REACT_APP_ENVIRONMENT === 'sidechain') {
+  if (process.env.REACT_APP_ENVIRONMENT === 'custom') {
```

Comment:
Since for the most part you use `custom network` together, I think it makes sense to keep that convention throughout
```suggestion
  if (process.env.REACT_APP_ENVIRONMENT === 'custom_network') {
```

Reply:
The URL will be `custom.xrpl.org`, which is why I kept the env value `custom`